stdlib/list.gr

```diff
@@ -132,15 +136,18 @@ let forEachi = (fn, list) => {
 let rec filter = (fn, list) => {
   match (list) {
     [] => [],
-    [first, ...rest] => if (fn(first)) [first, ...filter(fn, rest)] else filter(fn, rest)
+    [first, ...rest] =>
+      if (fn(first)) [first, ...filter(fn, rest)] else filter(fn, rest),
   }
 }
 
 let filteri = (fn, list) => {
   let rec iter = (fn, list, index) => {
     match (list) {
       [] => [],
-      [first, ...rest] => if (fn(first, index)) [first, ...iter(fn, rest, index + 1)] else iter(fn, rest, index + 1)
+      [first, ...rest] =>
+        if (fn(first, index)) [first, ...iter(fn, rest, index + 1)]
+        else iter(fn, rest, index + 1),
```

Comment:
We don't want to indent these, right? cc @ospencer

Reply:
I don't think so as if they were blocks the if and else would be where they are now